library/gradle

```diff
@@ -21,13 +21,13 @@ Tags: 6.8.3-jre11, 6.8.3-jre11-hotspot, 6.8-jre11, 6.8-jre11-hotspot, jre11, jre
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
 Directory: hotspot/jre11
 
-Tags: 6.8.3-jdk15, 6.8.3-jdk15-hotspot, 6.8-jdk15, 6.8-jdk15-hotspot, jdk15, jdk15-hotspot
+Tags: 6.8.3-jdk16, 6.8.3-jdk16-hotspot, 6.8-jdk16, 6.8-jdk16-hotspot, jdk16, jdk16-hotspot
```

Comment:
Shouldn't that read 7.0.0 rather than 6.8.3?

Reply:
Oops, yes. Thank you.